src/components/ContentSidebar/SidebarVersions.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @flow
+ * @file Versions sidebar component
+ * @author Box
+ */
+
+import React from 'react';
+import VersionHistoryLink from 'box-react-ui/lib/features/item-details/VersionHistoryLink';
+import type { FileVersions } from '../../flowTypes';
+
+type Props = {
+    onVersionHistoryClick?: Function,
+    versions: FileVersions
+};
+
+const SidebarVersions = ({ onVersionHistoryClick, versions }: Props) => {
+    const { total_count } = versions;
+
+    if (!total_count) {
+        return null;
+    }
+
+    return (
+        <section className='bcs-section'>
```

Comment:
The other commit from jeremy has some improvements of <SidebarSection> without need of a title, perhaps see if that works well here.

Reply:
👍